### PR TITLE
Update all new league Uniques

### PR DESCRIPTION
**Pull request by LocalIdentity**
Updates all the new league uniques with their stat ranges
Adds support for Corrosion from Ceaseless Feast gloves
Adds support for Enemy Grasping Vines from Inextricable Fate boots
Adds parsing for other minor mods

### CODE REVIEW

File: src/Data/Uniques/bow.lua [hunk 1]
```diff
@@ -220,6 +220,17 @@ Implicits: 2
...
 {variant:4}Gain 100% of Bow Physical Damage as Extra Damage of each Element
 ]],[[
+The Gluttonous Tide
+Citadel Bow
+League: Siege of the Atlas
+Source: Drops from unique{The Eater of Worlds}
+Requires Level 58, 185 Dex
+(120-160)% increased Physical Damage
+(16-20)% increased Attack Speed
+Lose all frenzy charges on reaching maximum frenzy charges to make the next bow attack you perform fire that many additional arrows
```
Comment (Nostrademous) [inline src/Data/Uniques/bow.lua:230]: Capitalization doesn't match poedb:
`Lose all Frenzy Charges on reaching Maximum Frenzy Charges to make the next Bow Attack you perform fire that many additional Arrows`

File: src/Data/Uniques/Special/Generated.lua [hunk 2]
```diff
@@ -559,6 +559,12 @@ function buildForbidden(classNotables)
 				index = index + 1
 			end
 		end
+		table.insert(forbidden[name], "League: Siege of the Atlas")
```
Comment (Wires77) [inline src/Data/Uniques/Special/Generated.lua:562]: This makes the wording become "Exclusive to", and Siege isn't the league, so it'll be available after this league too.

We should remove these lines so they match other boss drops like Watcher's Eye, Mark of the _____, Solstice Vigil, etc. (Goes for all the uniques in this PR)
image

File: src/Data/Uniques/amulet.lua [hunk 3]
```diff
@@ -55,6 +55,18 @@ Implicits: 1
 (80-100)% increased Rarity of Items found with a Normal Item equipped
 (10-15)% increased Quantity of Items found with a Magic Item equipped
 ]],[[
+Ashes of the Stars
+Onyx Amulet
+League: Siege of the Atlas
+Source: Drops from unique{The Eater of Worlds}
+Requires Level 61
```
Comment (Wires77) [inline src/Data/Uniques/amulet.lua:62]:
```suggestion
Requires Level 60
```

**Review by Wires77 — CHANGES_REQUESTED** [review]
Minor lvl requirement typo, and we should remove the "League" lines to stay consistent, but otherwise looks good

**Review by Wires77 — APPROVED** [review]

**Pull request merged**